tests/dummy/config/environment.js

```diff
@@ -11,6 +11,7 @@ module.exports = function(environment) {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
         // e.g. 'with-controller': true
+        'ember-glimmer-remove-application-template-wrapper': true,
```

Comment:
Lets remove this.

In order to test (until https://github.com/ember-cli/ember-try/pull/181 lands in a release) we will need to:

1. add an ember-try scenario for `ember-canary-without-application-wrapper`
2. add that scenario to .travis.yml
3. update `tests/dummy/config/environment.js` to do:

```js
EmberENV: {
  // ...snip other normal stuff...
  _APPLICATION_TEMPLATE_WRAPPER: process.env.EMBER_TRY_CURRENT_SCENARIO === 'ember-canary-without-application-wrapper' ? false : true
}
```